pkg/liqonet/ipam.go

```diff
@@ -648,3 +759,238 @@ func (liqoIPAM *IPAM) GetClusterExternalCIDR(mask uint8) (string, error) {
 	}
 	return externalCIDR, nil
 }
+
+func (liqoIPAM *IPAM) ipBelongsToPodCIDR(ip string) (bool, error) {
```

Comment:
Is it possible to add a comment to this method?

Reply:
Sure